subsys/bluetooth/mesh/transport.c

```diff
@@ -128,10 +128,17 @@ static struct seg_rx {
 				 ctl:1,
 				 in_use:1,
 				 obo:1;
-	uint8_t                     ttl;
+	uint32_t                    ttl: 8,
```

Comment:
There is no additional resource occupation here, because the structure needs to be aligned with 4 bytes, and there is a filling of 3 bytes before.